src/status_im2/contexts/onboarding/common/overlay/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.contexts.onboarding.common.overlay.view
+  (:require [react-native.reanimated :as reanimated]
+            [react-native.blur :as blur]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.onboarding.common.overlay.style :as style]
+            [status-im2.constants :as constants]))
+
+(def max-blur-amount 30)
+
+(defonce timer-interval (atom nil))
+(defonce blur-dismiss-fn-atom (atom nil))
+(defonce blur-show-fn-atom (atom nil))
+
+(defn f-view
+  [blur-amount]
+  (let [opacity                     (reanimated/use-shared-value 0)
+        transition-half-duration-ms (/ constants/onboarding-modal-animation-duration 2)
+        blur-show-fn                (fn []
+                                      (reanimated/animate opacity
+                                                          1
+                                                          transition-half-duration-ms)
+                                      (js/clearInterval @timer-interval)
+                                      (reset! timer-interval
+                                        (js/setInterval
+                                         (fn []
+                                           (if (< @blur-amount max-blur-amount)
+                                             (swap! blur-amount + 1)
+                                             (js/clearInterval @timer-interval)))
+                                         (/ constants/onboarding-modal-animation-duration
+                                            max-blur-amount
+                                            2))))
```

Comment:
We cannot animate `blur-amount`, so we use a ratom and a timer to update blur-amount value + animate opacity to achieve some smoothness.